README.md

```diff
@@ -30,27 +35,38 @@ When you export this image in python using pillow library (PIL.Image.open(file_n
 * 'trunk': [0, 124, 124]
 
 
-## 2- Presence Absence of Traits
+## 2- Default Tool: Presence
+
+The input of the presence class, or tool, is a semgemented image named "basename_segmented.png" generated by [Virginia Tech](https://github.com/hdr-bgnn/BGNN-trait-segmentation).
+
+The output of the presence tool is a presence table named "basename_presence.json" for each image.
+
+To check for the presence of traits, the tool:
 
-The approach that we use for checking presence and absence of traits is the following:
+  1. Isolates an individual trait (e.g., isolates the dorsal_fin)
+  2. Counts the number of blobs for that trait
+  3. Calculates the percentage of the area of the largest blob as a proportion of the area of the total blobs for that trait
 
-  1. Isolate indivual traits (e.g., isolate the dorsal_fin)
-  2. Count number of blobs
-  3. Calculate percentage of large blob
 
+## 3- Optional Tools: Landmark, Visualize, Metadata, and Measure
 
-## 3- Other Functions
+The input for the measure tool is a metadata file named "basename_metadata.json" generated by [Drexel](https://github.com/hdr-bgnn/drexel_metadata/tree/Thibault/gen_metadata_mini).
```

Comment:
So this refers to a branch in the drexel_metadata repository, which is almost certainly not what we want to do. Hence, the correct (in the sense of more canonical) answer is presumably different?

Reply:
I put in the link to just the repo. Sorry about that!